platform/broadcom/one-aboot.mk

```diff
@@ -1,6 +1,6 @@
 # sonic broadcom one image installer
 
-SONIC_ONE_ABOOT_IMAGE = sonic-aboot-broadcom.swi
+SONIC_ONE_ABOOT_IMAGE = sonic-aboot-broadcom$(if $(ENABLE_SYNCD_RPC),-rpc).swi
```

Comment:
for simplicity, I suggest leave appending -rpc to the user. The build system still builds the same file. sonic-aboot-broadcom.swi, user will then rename the file properly.